File: Python/book.py
class Book:
    def __init__(self, titre, auteur, prix):
        self.titre = titre
        self.auteur = auteur
        self.prix = prix

    def view(self):
        print("Titre :", self.titre)
        print("Auteur :", self.auteur)
        print("Prix : $", self.prix)

# Programme de test
livre = Book("Marius", "Marius", 30)
livre.view()

File: PHP/book.php
<?php

class Book {
    public $titre;
    public $auteur;
    public $prix;

    public function __construct($titre, $auteur, $prix) {
        $this->titre = $titre;
        $this->auteur = $auteur;
        $this->prix = $prix;
    }

    public function view() {
        echo "Titre : ". $this->titre."\n";
        echo "Auteur : ". $this->auteur."\n";
        echo "Prix : $ ". $this->prix."\n";
    }
}

// Programme de test
$livre = new Book("Marius", "Marius", 30);
$livre->view();

?>